content/reference/sql/server/troubleshooting.md

```diff
@@ -42,6 +42,11 @@ To run garbage collection, you should first stop your running server. It is not
 
 Online garbage collection is on the roadmap and we will be implementing it in the back half of 2022.
 
+Another potential cause is a commit-heavy workflow that uses a database design that is antagonistic to Dolt's structural sharing. We've written thoroughly about this [here](https://www.dolthub.com/blog/2020-05-13-dolt-commit-graph-and-structural-sharing/), but some examples include
+
+* Primary keys with random values
```

Comment:
I think you should give a reason why like:

* Primary keys with random values produce non-sequential chunks when changed. (Not actually sure why this is bad)